src/web3/addon/keystore.js

```diff
@@ -68,6 +68,16 @@ module.exports.isAccountLocked = (web3, { address }) => {
   return web3.currentProvider.isAccountLocked(address);
 };
 
+module.exports.sign = (web3, { msg, chainId, address }, cb) => {
```

Comment:
[MEDIUM] IMO You could not pass `chainId` as that value is possible to be fetch from web3 provider, therefore I would suggest to replace that by the inner calculation of it

Reply:
Actually, I will call it specifically "signAuthToken" so people don't use this function instead of the web3 official ones. It also doesn't support chainId atm.